i18n/ja/docusaurus-plugin-content-docs/current/unraid-os/release-notes/6.12.0.md

```
@@ -37,9 +37,9 @@ Unraid 6.12 と互換性がないことが判明しているプラグインが
 
 影響を受けるプラグインの一部は別の開発者に引き継がれています。**Apps** ページで代替プラグインを検索することをおすすめします。プラグイン固有の質問は、そのプラグインのサポートスレッドでお願いします。
 
-### 既知の問題 {/* #known-issues */}
+### 既知の問題 {/* #known-issues */} {/* #known-issues */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)